src/WebJobs.Script.WebHost/StandbyManager.cs

```diff
@@ -44,16 +43,16 @@ public static bool IsWarmUpRequest(HttpRequest request)
                 request.Path.StartsWithSegments(new PathString($"/api/{WarmUpAlternateRoute}")));
         }
 
-        public static void Initialize(ScriptHostConfiguration config)
+        public static void Initialize(ScriptHostConfiguration config, ILoggerFactory standbyLoggerFactory)
```

Comment:
Should this param just be called "loggerFactory"? What makes it a "standbyLoggerFactory"?

Reply:
I've refactored it so it actually takes an ILogger now.